Lib/axis.py

```diff
@@ -2192,6 +2192,8 @@ def __setitem__(self, index, value):
 
     def __setslice__(self, low, high, value):
         # Hack to prevent netCDF overflow error on 64-bit architectures
+        if(self.isUnlimited() and self.isTime() and (high >= Max32int)):
```

Comment:
@dnadeau4 do we really need `self.isTime()` ? The same issue would probably be true if it was an umlimited but non time axis. No?